lang/user/id.xml

```diff
@@ -53,46 +49,46 @@
 	</about>
 
 	<app_context>
-		<check_app_state>Check Aplikasi state</check_app_state>
-		<copy_vita3k_summary>Copy Ringkasan Vita3K</copy_vita3k_summary>
-		<open_state_report>Buka state laporan</open_state_report>
+		<check_app_state>Cek state Aplikasi</check_app_state>
+		<copy_vita3k_summary>Salin Ringkasan Vita3K</copy_vita3k_summary>
+		<open_state_report>Buka laporan state</open_state_report>
 		<create_state_report>Bikin state laporan</create_state_report>
-		<copy_app_info>Copy Aplikasi Info</copy_app_info>
-		<name_and_id>Nama Dan Judul ID</name_and_id>
-		<app_summary>Ringkasan Aplikasi</app_summary>
+		<update_database>Perbarui database</update_database>
+		<copy_app_info>Salin info Apl</copy_app_info>
+		<name_and_id>Nama dan Judul ID</name_and_id>
+		<app_summary>Ringkasan Apl</app_summary>
 		<custom_config>Kastom Config</custom_config>
 		<create>Bikin</create>
```

Comment:
I think the word "Buat" is more appropriate here. "Bikin" is an informal word.

Reply:
Just make it better nowadays

Reply:
Bikin Just make it better nowadays